src/World.h

```diff
@@ -79,7 +79,7 @@ typedef cItemCallback<cFlowerPotEntity>    cFlowerPotCallback;
 
 
 
-
+const int CHUNK_CHANCE_ACCURACY = 100000;
```

Comment:
Also, the random unload scheme is a rather balanced heuristic. Consider a case where a player is running around in a ring, LRU would be extreme bad. The random scheme seems to not have any worst case scenarios.